src/ripple/protocol/Feature.h

```diff
@@ -21,26 +21,86 @@
 #define RIPPLE_PROTOCOL_FEATURE_H_INCLUDED
 
 #include <ripple/basics/base_uint.h>
+#include <boost/container/flat_map.hpp>
 #include <boost/optional.hpp>
 #include <array>
 #include <bitset>
 #include <string>
 
+/**
+ * @page Feature How to add new features
+ *
+ * Steps required to add new features to the code:
+ *
+ * 1) add the new feature name to the featureNames array below
+ * 2) add a uint256 declaration for the feature to the bottom of this file
+ * 3) add a uint256 definition for the feature to the corresponding source
+ *    file (Feature.cpp)
+ * 4) if the feature is going to be supported immediately, add its sha512half
+ *    value and name (matching exactly the featureName here) to the
+ *    supportedAmendments in Amendments.cpp. Even if the feature is not
+ *    immediately supported, consider adding it there and commenting out, to
+ *    simplify future enabling.
```

Comment:
FWIW, some Features are never intended to become Amendments.  Case in point are `CompareFlowV1V2 ` and `CompareTakerFlowCross `.  Consider this for your last point...
```
4) if the feature is going to be a supported Amendment in the near future,  add...
```